resources/js/frontend/index.js

```diff
@@ -31,22 +29,22 @@ const label = decodeEntities(settings.title) || labels.defaultLabel;
 let formSubmittedAlready = false;
 const Content = (props) => {
     const {eventRegistration, emitResponse} = props;
-    const {onPaymentSetup, onCheckoutValidation, onShippingRateSelectSuccess} = eventRegistration;
+    const {onPaymentSetup, onCheckoutValidation} = eventRegistration;
 
     jQuery('.wc-block-components-checkout-place-order-button').show();
 
     useEffect(() => {
-        let newAmount = null;
-
-        const unsubscribeFromShippingEvent = onShippingRateSelectSuccess(async () => {
-            const { total_price: currentTotalPrice } = cart.getCartTotals();
-
-            newAmount = Number(currentTotalPrice / 100).toFixed(2);
-        });
-
```

Comment:
@andryes @FabianaRBarros @Danyl-Kushnaryov This code above was a fix for something else?

Reply:
@jpcostello @anthonyvallone-paydock  All code written to fix this bug: https://paydock.atlassian.net/browse/SDK-592, has been removed in this PR

Reply:
@anthonyvallone-paydock I raised this PR from your branch - some code may still need to be merged based on comments. Please take a look.

Reply:
@Danyl-Kushnaryov i just branched off the most up to date power_board branch and made my changes.

Reply:
The removed part above is present in the power_board branch:

https://github.com/PayDock/e-commerce-woo/blob/power_board/resources/js/frontend/index.js#L39-L45